### DISCUSSION TITLE
Mixing HTML and Avalonia View inside WebBrowser

### DISCUSSION THREAD

**Discussion by npolyak** [AvaloniaUI/Avalonia#13659] [Q&A]
I have trouble making very simple Avalonia view appear under some HTML. Here is the code: 
https://github.com/npolyak/sketches/tree/main/NP.MixedHtmlAndAvaPage

I created a default Avalonia 11.5 project simplified it by removing the view models and placing a hardcoded text within the main view. 

Then I modified the index.html file to have Hello World heading on top of the <div id="out"/> (the div that is replaced with the MainView)

It appears fine before the main view is loaded, but the main view actually takes the whole window space so that I cannot display html above or next to it. 

Please help

**Comment by timunie** [comment]
@npolyak looks like one needs to adjust the app.cs file. I played with it a bit and managed to add some withe space around. But I'm not good enough in the css stuff to explain the details.  

```css
#out {
    position: relative;
    height: 80%;
    width: 80%;
}
```

**Reply by npolyak** [reply]
Thanks Timunie, I just do not understand why something needs to be done in CSS and why it just would not work as intended. Is there anyway for the Avalonia view to take only the space that is given to it within HTML?

**Reply by npolyak** [reply]
BTW, I tried adding the CSS code to app.css and the Avalonia View still takes the whole screen vertically. Can you create a full application that works or perhaps create a branch to my repository for that.

**Reply by timunie** [reply]
Sorry I have had no luck either to run your sample. I believe it has to do with the 100hv but that's all I can tell here.

**Comment by npolyak** [comment]
I created a new Avalonia issue https://github.com/AvaloniaUI/Avalonia/issues/13713
